### PR TITLE
Import badge lengths are set correctly

### PR DESCRIPTION
**Pull request by roonyh**
Import number background is adjusted correctly when number of imports grows

### CODE REVIEW

File: modules/web/js/ballerina/visitors/sizing-utils.js [hunk 1]
```diff
@@ -74,6 +74,16 @@ class SizingUtil {
         };
     }
 
+    getOnlyTextWidth(text, options={}) {
```
Comment (hemikak) [inline modules/web/js/ballerina/visitors/sizing-utils.js]: Shall we have js doc comments to this func. ?

Comment (roonyh) [inline modules/web/js/ballerina/visitors/sizing-utils.js]: Added in #1971

File: modules/web/js/ballerina/components/import-declaration.jsx [hunk 2]
```diff
@@ -19,33 +19,58 @@
...
+        const importNoFontSize = 13;
+        const noOfImportsLeftPadding = 12;
+        const iconLeftPadding = 12;
+        const noOfImportsBGHeight = 18;
+        const importLabelWidth = 48.37;
+
+        const noOfImports = this.props.imports.length;
+
+        const noOfImportsTextWidth = SizingUtils.getOnlyTextWidth(noOfImports, {fontSize: importNoFontSize});
+        const noOfImportsBGWidth = Math.max(noOfImportsTextWidth + 6, noOfImportsBGHeight);
```
Comment (sumuditha-viraj) [inline modules/web/js/ballerina/components/import-declaration.jsx]: Can we remove magic numbers? (Eg: 6)

Comment (roonyh) [inline modules/web/js/ballerina/components/import-declaration.jsx]: Done in #2006

**Pull request merged**